packages/legend-application-studio/src/stores/StoreRelational_LegendStudioApplicationPlugin_Extension.ts

```diff
@@ -97,4 +120,35 @@ export interface StoreRelational_LegendStudioApplicationPlugin_Extension
    * Get the list of renderers for the editor for a relational database authentication strategy.
    */
   getExtraAuthenticationStrategyEditorRenderers?(): AuthenticationStrategyEditorRenderer[];
+
+  // --------------------- relational database connection post-processor ------------------
+
+  /**
+   * Get the list of the supported type specifiers for post-processors
+   *
+   */
+  getExtraPostProcessorClassifiers?(): string[];
+
+  /**
+   * Get the list of classifiers for a post-processor specification.
+   */
+  getExtraPostProcessorClassifierGetters?(): PostProcessorTypeGetter[];
+
+  /**
+   * Get the list of creators for post-processor specification given the type specifier.
+   */
+  getExtraPostProcessorCreators?(
+    connectionValueState: RelationalDatabaseConnectionValueState,
+    observerContext: ObserverContext,
+  ): PostProcessorCreator[];
+
+  /**
+   * Get the list of states for a post-processor specification.
+   */
+  getextraPostProcessorStates?(): PostProcessorState[];
```

Comment:
--> `getExtraPostProcessorStateCreators?(): PostProcessorStateCreator[];` Please update the description accordingly